std/experimental/ndslice/slice.d

```diff
@@ -1743,6 +1744,155 @@ struct Slice(size_t _N, _Range)
         assert(a != [[9, 2], [3, 4]]);
     }
 
+    /++
+    Computes hash value using MurmurHash3 algorithms without finalization step.
+    Hash map implementations should care about finalization step.
+    Built-in associative arrays has finalization step.
+
+    See_also: $(ZREF2 Slice.toMurmurHash3, toMurmurHash3), $(MREF std, _digest, murmurhash)
+    +/
+    pure nothrow @nogc
+    size_t toHash()() const @safe
```

Comment:
Why is it a template ?
And if it is, it shouldn't have explicit attribute but `const`.